omsApp/src/MAX_trajectoryScan.st

```diff
@@ -1183,7 +1183,10 @@ static double rawToUser(int raw, double off, int dir, double res) {
 
 /**************************************************************************************/
 static int loadTrajectory(SS_ID ssId, struct UserVar *pVar, int simMode) {
-	int i, j, k, n, currUpdateRate;
+	int i, j, k, n;
+#if 0
+        int currUpdateRate;
+#endif
```

Comment:
With modern version control systems seeing an `#if 0` is always a bad code smell. If it isn't used, remove it. If the code is a debug aid, make it conditional on some macro which makes that obvious and easy to turn them all on/off at once.

Reply:
Sorry, the #if 0 was a left over I failed to clean up. Will do.

Reply:
It's not my module to decide, but I do generally recommend removing code that is never built. If someone wants the old code back they can ask git for it.

Reply:
I'm ok with the removal of this code.